ledger/src/blockstore_db.rs

```diff
@@ -1532,8 +1533,15 @@ fn get_db_options(access_type: &AccessType) -> Options {
     options
 }
 
-fn excludes_from_compaction(cf_name: &str) -> bool {
-    // list of Column Families must be excluded from compaction:
+// Returns whether automatic compactions should be disabled based upon access type
+fn disable_auto_compactions(access_type: &AccessType) -> bool {
+    // Disable automatic compactions in maintenance mode to prevent accidental cleaning
+    matches!(access_type, AccessType::PrimaryOnlyForMaintenance)
+}
+
+// Returns whether the supplied column (name) should be excluded from compaction
+fn exclude_column_from_compaction(cf_name: &str) -> bool {
```

Comment:
`excludes_from_compaction` ==> `exclude_column_from_compaction`

Reply:
nit: maybe `is_excluded_from_compaction` as it returns a bool?

Reply:
I like it; in a similar vein, I also changed `disable_auto_compactions` to `should_disable_auto_compactions`